### ISSUE TITLE
Store Public Keys

### ISSUE THREAD

**Issue by sirvan3tr** [farridav/keymanager#6]
You can expand this to have an option where you can store the public keys somewhere where they can be retrieved by a user with a given permission, a group of developers thus would have access to each others public keys that they can use when they need to.

**Comment by farridav** [comment]
Yeah this is a good idea, I had actually already started work on this, storing them In a yaml file for easier massive user/server management.. will update this issue
